### PR TITLE
127 api baseline

### PR DESCRIPTION
**Pull request by BeckerFrank**
Here my try to fix the #127

### CODE REVIEW

File: org.eclipse.mylyn.releng/oomph/Mylyn.setup [hunk 1]
```diff
@@ -92,10 +94,60 @@
...
+        <repositoryList
+            name="latest">
+          <repository
+              url="https://download.eclipse.org/mylyn/releases/3.26"/>
+          <repository
+              url="https://download.eclipse.org/egit/updates-6.2"/>
+          <repository
+              url="https://download.eclipse.org/tools/cdt/releases/11.0"/>
+          <repository
+              url="https://download.eclipse.org/eclipse/updates/4.27/R-4.27-202303020300"/>
```
Comment (ruspl-afed) [inline org.eclipse.mylyn.releng/oomph/Mylyn.setup:143]: actually we had Eclipse Platform 4.26 in our target, it is better to use the target from the latest release to define the baseline for upcoming one

File: org.eclipse.mylyn.releng/oomph/Mylyn.setup [hunk 2]
```diff
@@ -92,10 +94,60 @@
     <setupTask
```
Comment (ruspl-afed) [inline org.eclipse.mylyn.releng/oomph/Mylyn.setup:94]: Do we need this task? If not, it is better to remove it

File: org.eclipse.mylyn.releng/oomph/Mylyn.setup [hunk 3]
```diff
@@ -92,10 +94,60 @@
...
         xsi:type="pde:APIBaselineTask"
         id="mylyn.baseline"
+        disabled="true"
```
Comment (ruspl-afed) [inline org.eclipse.mylyn.releng/oomph/Mylyn.setup:97]: Do we need this task? If not, it is better to remove it

**Review by ruspl-afed — APPROVED** [review]
LGTM, some minor comments

**Comment by BeckerFrank** [comment]
@ruspl-afed: for some reason I can not merge this. Can you try?

**Comment by ruspl-afed** [comment]
done

**Pull request merged**